### PR TITLE
4 GET /api/reviews/:review_id complete with error tests

### PR DESCRIPTION
**Pull request by Oathian**

### CODE REVIEW

File: models/getReviewByIdModel.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return db.query(`SELECT * FROM reviews WHERE review_id = $1;`,[review_id])
+    .then(({ rows }) => {
+
+        if(rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "Resource not found"});
+        }
+
+        if(rows[0].review_img_url) {
+            rows[0].review_img_url = "https://images.pexels.com/photos/163064/play-stone-network-networked-interactive-163064.jpeg"
+        };
```
Comment (daviidbartlett) [inline models/getReviewByIdModel.js]: what's the motivation behind this line?

File: models/getReviewByIdModel.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        
+        if(rows[0].votes) {
+            rows[0].votes = 0;
+        };
```
Comment (daviidbartlett) [inline models/getReviewByIdModel.js]: and this?

File: models/getReviewByIdModel.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+        if(rows[0].created_at) {
+            rows[0].created_at = Date.now();
+        };
```
Comment (daviidbartlett) [inline models/getReviewByIdModel.js]: and this?

**Comment by DCrawley94** [comment]
Looking good! Feel free to merge :)

**Pull request merged**